### ISSUE TITLE
Train/Val/Test set split

### ISSUE THREAD

**Issue by XiChen-97** [uzh-rpg/aegnn#14]
Hi @simon-schaefer , thanks for your impressive work! When I tried to train the model, I was wondering how is the train/val/test set split and how is the testing result reported in the experiment? Could you please give some hints here? 

e.g. 1) for the N-cars/Ncaltech/Gen dataset, what is the portion of train/val/test set respectively? 
2) How is the testing performance reported? Is it based on the validation set or using cross validation etc.

Thanks in advance and look forward to your response.

**Comment by simon-schaefer** [comment]
Hi XiChen-97, 

thank you for your interest in our work. The train/val/test split was done randomly, with 80-10-10 split size when I remember correctly. The testing performance is just the model accuracy on the test set, i.e. without using cross validation. 

Best regards, 
Simon
